app/kubemci/pkg/gcp/urlmap/urlmapsyncer.go

```diff
@@ -117,6 +118,32 @@ func (s *URLMapSyncer) DeleteURLMap() error {
 	}
 }
 
+// See interface for comment.
+func (s *URLMapSyncer) RemoveClustersFromStatus(clusters []string) error {
+	fmt.Println("Removing clusters", clusters, "from url map")
```

Comment:
Should this use `Printf`?

Reply:
um why? We use Println throughout the code base for simple output strings that dont require special formatting.